frontend/src/components/widgets/inputs/NumberInput.vue

```diff
@@ -210,12 +219,24 @@ export default defineComponent({
 		onIncrement(direction: IncrementDirection) {
 			if (Number.isNaN(this.value)) return;
 
-			if (this.stepIsMultiplier) {
-				const directionMultiplier = direction === IncrementDirection.Increase ? this.step : 1 / this.step;
-				this.updateValue(this.value * directionMultiplier);
-			} else {
-				const directionAddend = direction === IncrementDirection.Increase ? this.step : -this.step;
-				this.updateValue(this.value + directionAddend);
+			switch (this.incrementBehavior) {
+				case IncrementBehavior.Add: {
+					const directionAddend = direction === IncrementDirection.Increase ? this.incrementFactor : -this.incrementFactor;
```

Comment:
(direction as number * 2 - 1) * this.IncrementFactor if you want to be cool

Reply:
`direction` is an enum, which is the string `"Increase"` or `"Decrease"`.